gradle/libs.versions.toml

```diff
@@ -100,6 +102,11 @@ xml-jb-annotations = { module = "org.jetbrains:annotations", version = "24.0.1"
 # GIT
 git-jgit = { module = "org.eclipse.jgit:org.eclipse.jgit", version = "3.7.1.201504261725-r" }
 
+# Room
+room-runtime = { group = "androidx.room", name = "room-runtime", version.ref = "room" }
+room-compiler = { group = "androidx.room", name = "room-compiler", version.ref = "room" }
+room-ktx = { group = "androidx.room", name = "room-ktx", version.ref = "room" }
```

Comment:
These should be renamed to `androidx-room-*` to match other AndroidX dependency names in the catalog.